tests/controls/maskededittest.cpp

```diff
@@ -0,0 +1,149 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        tests/controls/maskededittest.cpp
+// Purpose:     wxMaskedEdit controls unit test
+// Author:      Manuel Martin
+// Created:     2012-07-30
+// RCS-ID:      $Id:
+// Copyright:   (c)
+///////////////////////////////////////////////////////////////////////////////
+
+#include "testprec.h"
+
+#if wxUSE_MASKED_EDIT
+
+#ifdef __BORLANDC__
+    #pragma hdrstop
+#endif
+
+#ifndef WX_PRECOMP
+    #include "wx/app.h"
+#endif // WX_PRECOMP
+
+#include "wx/maskededit.h"
+
+#include "testableframe.h"
+#include "wx/uiaction.h"
+
+class MaskedEditTestCase : public CppUnit::TestCase
```

Comment:
It would be nice to avoid using CppUnit compatibility stuff in the new code.

Reply:
@vadz ,
New test should use "catch", right?
I will see what should be done...